<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="data_enter" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>0</y>
      <width>261</width>
      <height>211</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLineEdit" name="Name_edit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Name</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLineEdit" name="Phone_edit">
         <property name="text">
          <string>(555)555-5555</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Phone Number</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLineEdit" name="Address_edit"/>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Address</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLineEdit" name="Email_edit"/>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>E-Mail</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLineEdit" name="Web_edit"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Website address</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="status_label">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="submit_data">
         <property name="text">
          <string>Submit</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="exit_data">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>193</width>
      <height>173</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="init_menu">
     <item>
      <widget class="QPushButton" name="enter_data">
       <property name="text">
        <string>Enter Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="view_data">
       <property name="text">
        <string>View Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="search_data">
       <property name="text">
        <string>Search Data</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QTableWidget" name="data_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>220</y>
      <width>581</width>
      <height>251</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustIgnored</enum>
    </property>
    <attribute name="horizontalHeaderVisible">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderCascadingSectionResizes">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderDefaultSectionSize">
     <number>115</number>
    </attribute>
    <attribute name="horizontalHeaderHighlightSections">
     <bool>true</bool>
    </attribute>
    <attribute name="horizontalHeaderMinimumSectionSize">
     <number>21</number>
    </attribute>
    <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
     <bool>false</bool>
    </attribute>
    <attribute name="horizontalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <attribute name="verticalHeaderStretchLastSection">
     <bool>false</bool>
    </attribute>
    <column>
     <property name="text">
      <string>Name</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Phone Number</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Address</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>E-Mail</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Website</string>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="search_widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>380</y>
      <width>230</width>
      <height>91</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_4">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="search_label">
         <property name="text">
          <string>Name Search:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="search_name"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="search_status">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QPushButton" name="search_ok">
         <property name="text">
          <string>Ok</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="search_exit">
         <property name="text">
          <string>Exit</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="load_prompt" native="true">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>160</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QPushButton" name="load_auto">
       <property name="text">
        <string>Load Data</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="load_manual">
       <property name="text">
        <string>Manual Enter</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="data_save">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>180</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Data</string>
    </property>
   </widget>
   <zorder>verticalLayoutWidget_2</zorder>
   <zorder>data_enter</zorder>
   <zorder>data_table</zorder>
   <zorder>search_widget</zorder>
   <zorder>load_prompt</zorder>
   <zorder>data_save</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>717</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>enter_data</sender>
   <signal>clicked()</signal>
   <receiver>load_prompt</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>submit_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>data_submit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>360</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>196</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>data_exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>545</x>
     <y>228</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>view_data</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>view_data()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>119</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>338</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_data</sender>
   <signal>clicked()</signal>
   <receiver>search_widget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>218</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_exit</sender>
   <signal>clicked()</signal>
   <receiver>search_widget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>212</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>154</x>
     <y>460</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>search_ok</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>data_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>488</y>
    </hint>
    <hint type="destinationlabel">
     <x>532</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_manual</sender>
   <signal>clicked()</signal>
   <receiver>data_enter</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>480</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_manual</sender>
   <signal>clicked()</signal>
   <receiver>load_prompt</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>309</x>
     <y>152</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>134</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>load_auto</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>data_load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>data_save</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>data_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>230</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>265</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>data_submit()</slot>
  <slot>data_exit()</slot>
  <slot>view_data()</slot>
  <slot>data_search()</slot>
  <slot>data_load()</slot>
  <slot>data_save()</slot>
 </slots>
</ui>
